server/market/market.go

```diff
@@ -1258,29 +1266,27 @@ func (m *Market) enqueueEpoch(eq *epochPump, epoch *EpochQueue) bool {
 
 	// Start preimage collection.
 	go func() {
-		rq.cSum, rq.ordersRevealed, rq.misses = m.epochStart(orders)
+		rq.cSum, rq.ordersRevealed, rq.misses = m.epochStart(orders, epoch.End)
 		close(rq.ready)
 	}()
 
 	return true
 }
 
 // epochStart collects order preimages, and penalizes users who fail to respond.
-func (m *Market) epochStart(orders []order.Order) (cSum []byte, ordersRevealed []*matcher.OrderRevealed, misses []order.Order) {
+func (m *Market) epochStart(orders []order.Order, closeTime time.Time) (cSum []byte, ordersRevealed []*matcher.OrderRevealed, misses []order.Order) {
```

Comment:
Just a nit, but I'm noticing you're using `closeTime` here, but `epochEnd` in `(*AuthManager).MissedPreimage`, as well as using `epoch.End` as the argument to this function the only place it's used. I'm certain they mean the same thing, so they might as well have the same name?